test/integration-tests/ui/PackageDependencyProvider.test.ts

```diff
@@ -0,0 +1,118 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the VS Code Swift open source project
+//
+// Copyright (c) 2024 the VS Code Swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of VS Code Swift project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import { expect } from "chai";
+import {
+    PackageDependenciesProvider,
+    PackageNode,
+} from "../../../src/ui/PackageDependencyProvider";
+import { folderContextPromise, globalWorkspaceContextPromise } from "../extension.test";
+import { executeTaskAndWaitForResult, waitForNoRunningTasks } from "../../utilities";
+import { getBuildAllTask, SwiftTask } from "../../../src/tasks/SwiftTaskProvider";
+import { testAssetPath } from "../../fixtures";
+import { Version } from "../../../src/utilities/version";
+
+suite("PackageDependencyProvider Test Suite", function () {
+    let treeProvider: PackageDependenciesProvider;
+    suiteSetup(async function () {
+        this.timeout(2 * 60 * 1000); // Allow up to 2 minutes to build
```

Comment:
nit, other tests set the timeout at the top level using `this.timeout(...)` instead of inside a suiteSetup

Reply:
done